sdk/digitaltwins/digital-twins-core/review/digital-twins-core.api.md

```diff
@@ -334,6 +255,12 @@ export interface IncomingRelationshipCollection {
     value?: IncomingRelationship[];
 }
 
+// @public
+export interface PagedDigitalTwinsModelDataCollection {
+    nextLink?: string;
```

Comment:
Are developers supposed to understand how to use the nextLink directly? I don't see any way to pass it in via `PageSettings`, so I'm not sure how this would be used by the SDK.

In Search we encoded this into an opaque token they could pass back to us: https://github.com/Azure/azure-sdk-for-js/blob/20072d2a34884dae1c16d8f6d96097c2830bc6d8/sdk/search/search-documents/src/searchClient.ts#L261

https://github.com/Azure/azure-sdk-for-js/blob/20072d2a34884dae1c16d8f6d96097c2830bc6d8/sdk/search/search-documents/src/indexModels.ts#L108

Reply:
I see your point. I would go ahead and release this preview for the bug bash.